docs/running-e3sm-guide/guide-long-term-archiving.md

```diff
@@ -0,0 +1,228 @@
+# Long-Term Archiving
+
+Simulations that are deemed sufficiently valuable should be archived using `zstash` for long-term preservation. You can ask questions about `zstash` on the [zstash discussion board](https://github.com/E3SM-Project/zstash/discussions/categories/questions).
+
+!!! warning
+    Compy, Anvil and Chrysalis do not have local HPSS. We rely on NERSC HPSS for long-term archiving. If you are archiving a simulation run on Compy or LCRC (Chrysalis/Anvil), do all of the following steps. If you are archiving a simulation run on NERSC (Perlmutter), skip to step 4.
```

Comment:
Can we just skip to step 4? It seems like step 1 should be done on all machines.

Reply:
it doesn't seem like "skipping to step 4" is what we want to say here. I would just delete this sentence. If the user is already on NERSC, then they should have the  common sense to realize they don't need to transfer from NERSC to NERSC. But maybe we can clarify down in step 2 about how the user should transfer the data to NERSC if needed. Maybe that could be put as "step 2.1" or an optional "step 3" to be even clearer that it is a distinct step in the process, but not always required.